mule-extensions-api/src/main/java/org/mule/runtime/extension/api/declaration/type/ExtensionsObjectFieldHandler.java

```diff
@@ -383,4 +436,43 @@ private void optionalField(ObjectFieldTypeBuilder fieldBuilder, String defaultVa
       fieldBuilder.with(new DefaultValueAnnotation(defaultValue));
     }
   }
+
+  private <R extends Annotation, S extends Annotation, T> T getAnnotationValueFromField(
```

Comment:
I would move this method and all calls to it to a utils class, so that we remove all references to the annotations in this class and make calls to it like isText(---) getExampleValue(----) getPlacementeInfo(---)